onnxruntime/contrib_ops/cuda/quantization/qorder_attention.cc

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "qorder_attention.h"
+#include "contrib_ops/cuda/bert/attention_impl.h"
+#include "core/providers/cuda/cuda_common.h"
+#include "core/providers/cuda/shared_inc/fpgeneric.h"
+
+using namespace onnxruntime::cuda;
+using namespace ::onnxruntime::common;
+using namespace ONNX_NAMESPACE;
+
+namespace onnxruntime {
+namespace contrib {
+namespace cuda {
+
+#define REGISTER_KERNEL_TYPED(T)                                  \
+  ONNX_OPERATOR_TYPED_KERNEL_EX(                                  \
+      QOrderedAttention,                                          \
+      kMSDomain,                                                  \
+      1,                                                          \
+      T,                                                          \
+      kCudaExecutionProvider,                                     \
+      (*KernelDefBuilder::Create())                               \
+          .TypeConstraint("T", DataTypeImpl::GetTensorType<T>()), \
+      QOrderedAttention);
+
+REGISTER_KERNEL_TYPED(int8_t)
+
+QOrderedAttention::QOrderedAttention(const OpKernelInfo& info) : CudaKernel(info), AttentionBase(info) {
+  order_input_ = GetCublasLtOrderAttr(info, "order_input");
+  order_weight_ = GetCublasLtOrderAttr(info, "order_weight");
+  order_bias_ = GetCublasLtOrderAttr(info, "order_bias");
+  order_output_ = GetCublasLtOrderAttr(info, "order_output");
+  //TODO: any checks?
+  // bugbug
```

Comment:
question: do we need to check the order? would there be other order combinations in the future in onnx model?